### PR TITLE
Set aws credentials os env before change home dir

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: rosa/cli.py [hunk 1]
```diff
@@ -270,6 +273,28 @@ def build_execute_command(command, allowed_commands=None, aws_region=None):
     return execute_command(command=command)
 
 
+def aws_credentials():
```
Comment (oharan2) [inline rosa/cli.py:276]: As for how it's implemented here, do you think we should move it to `cloud_tools` (for general use) and call it something like `set_aws_credentials `?

**Review by omrirh — APPROVED** [review]

File: rosa/cli.py (hunk 1 above)
Comment (myakove) [reply rosa/cli.py:276]: Make sense
https://github.com/RedHatQE/cloud-tools/pull/80

**Comment by myakove** [comment]
depends on https://github.com/RedHatQE/cloud-tools/pull/80

**Review by rnetser — APPROVED** [review]

**Review by oharan2 — APPROVED** [review]

**Pull request merged**